### ISSUE TITLE
Bug on Samsung Note 10

### ISSUE THREAD

**Issue by kekedaine** [Baseflow/flutter-geolocator#327]
i use this command to get location
`Position position = await Geolocator()
          .getCurrentPosition(desiredAccuracy: LocationAccuracy.low);`
But with Samsung Note 10, it does not work. And not run next command and not throw any exception.

**Comment by ChiefMax** [comment]
Hey @kekedaine, could you please give us some more information about this bug?

**Comment by FlashJonas** [comment]
I have the same issue on Galaxy note 8.

```
Future getFirstLocation() async {
    Position position;

    print("getfirstlocation");

    GeolocationStatus geolocationStatus  = await Geolocator().checkGeolocationPermissionStatus();

    print("geolocationStatus: ${geolocationStatus}");

    var temp = Geolocator()
        .getCurrentPosition(desiredAccuracy: LocationAccuracy.lowest).catchError((e)=> {print("then")});

    print("temp= $temp");

    try {
      position = await Geolocator()
          .getCurrentPosition(desiredAccuracy: LocationAccuracy.high);
    } catch (error) {
      print("getfirstlocation error 1");
      try {
        position = await Geolocator()
            .getCurrentPosition(desiredAccuracy: LocationAccuracy.high);
      } catch (error) {
        print("error in geolocation $error");
      }
    }

    print("getfirstlocation position: $position");
```
prints 
> getfirstlocation
> geolocationStatus: GeolocationStatus.granted
> temp= Instance of 'Future<Position>'

No errors are thrown nor catched.
The Future seems to never resolve nor fail.
No code further down in the function is called (which of course breaks my app).

**Comment by FlashJonas** [comment]
Switching out geolocator: ^5.1.3 for the alternative library:
location: ^2.3.5
(and updating the code of course)
works on Galaxy Note8 as a desperate fix while this is being solved.

**Comment by StijnWoerkom** [comment]
@FlashJonas Did you try to run the app on an emulator? You could do this to test if the bug depends on the device or the code itself.

**Comment by synw** [comment]
Same here on Note 4: the example always return nulls only on that device, it works on my others Android devices

**Comment by njovy** [comment]
@StijnWoerkom  This bug still appears in the latest plugin.

**Comment by BrunoJunior** [comment]
I have the same issue on Android emulator ...
I try to print the position when a screen is initialized.
There is my code :
```dart
class _LoadingScreenState extends State<LoadingScreen> {
  void printPosition() async {
    GeolocationStatus geolocationStatus =
        await Geolocator().checkGeolocationPermissionStatus();
    print(geolocationStatus);
    Position position = await Geolocator()
        .getCurrentPosition(desiredAccuracy: LocationAccuracy.low);
    print(position);
  }

  @override
  void initState() {
    print('init');
    super.initState();
    printPosition();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold();
  }
}
```

And the console result :

> I/flutter ( 6254): init
> I/flutter ( 6254): GeolocationStatus.granted

**Comment by mvanbeusekom** [comment]
This bug is solved with version 6.0.0 of the geolocator plugin.
